Userland/Libraries/LibGUI/Statusbar.cpp

```diff
@@ -38,6 +38,17 @@ NonnullRefPtr<Statusbar::Segment> Statusbar::create_segment()
     return widget;
 }
 
+void Statusbar::child_event(Core::ChildEvent& event)
```

Comment:
This is mysterious enough that we should have a comment explaining what we're doing and why.

Reply:
I put the comment in the function body, not sure if that's a good place for it. But if someone is confused by what this function does they will look at it.